### ISSUE TITLE
Everything Bagel's next steps

### ISSUE THREAD

**Issue by rmoff** [treeverse/lakeFS#6077]
## tl;dr

Everything Bagel's spiritual home is now https://github.com/treeverse/lakeFS-samples, from where it is more discoverable and usable. 

I would now like to tidy up the [existing Everything Bagel code](https://github.com/treeverse/lakeFS/blob/master/deployments/compose/) in lakeFS repo and looking for opinions on the approach to take.

### lakeFS-samples

lakeFS-samples was [updated recently](https://github.com/treeverse/lakeFS-samples/pull/74) to be built around a [single Docker Compose](https://github.com/treeverse/lakeFS-samples/blob/main/docker-compose.yml) with a [library of notebooks](https://github.com/treeverse/lakeFS-samples/blob/main/notebooks/00_index.ipynb) for the user to explore. 

![image](https://github.com/treeverse/lakeFS/assets/3671582/6678e9d8-0471-48c9-ab71-598dacd82b22)

Some examples don't easily fit in this model and are included in the repository [standalone](https://github.com/treeverse/lakeFS-samples/tree/main/standalone). These include those with other dependencies (e.g. Airflow) or more complex deployments that would take too long to migrate. 

The background and discussion for this is [here](https://github.com/treeverse/devex-internal/issues/262#issuecomment-1554546627) and on Slack.

### Everything Bagel

https://github.com/treeverse/lakeFS/blob/master/deployments/compose/

![image](https://github.com/treeverse/lakeFS/assets/3671582/1a7f979a-5189-4f31-9494-13035be322da)

Everything Bagel includes things not present in lakeFS-samples: 

* Trino
* Catalog (Hive)
* dbt
* [Standalone spark containers, but Spark is included in the Jupyter notebook] 

👉 **However, these are not referenced in any docs or samples**. One [Everything Bagel blog](https://lakefs.io/blog/the-everything-bagel-ii-versioned-data-lake-tables-with-lakefs-and-trino/) (41 page views in 90 days) covers Trino but doesn't clearly send users to EB. 

The docs cover these tools (e.g. [Trino](https://docs.lakefs.io/integrations/presto_trino.html), [dbt](https://docs.lakefs.io/integrations/dbt.html)) but don't reference or link to Everything Bagel. 

**Retaining Everything Bagel, or porting these containers, for the sake of it seems redundant. How will users know to use them?** 

## Proposal

The driving aim of all this is to consolidate our demo code into one place so that users can easily find and run it. 

Therefore, I suggest that we do the following (split into parts to avoid blocking/big bang): 

1. Stage 1 - remove Everything Bagel
    1. Update both EB blogs ([1](https://lakefs.io/blog/the-docker-everything-bagel-spin-up-a-local-data-stack/) [2](https://lakefs.io/blog/the-everything-bagel-ii-versioned-data-lake-tables-with-lakefs-and-trino/)) to point users to https://github.com/treeverse/lakeFS-samples
    1. `rm -rf ./deployments/compose/*`
    1. Update `./deployments/compose/README.md` to point users to https://github.com/treeverse/lakeFS-samples

2. Stage 2 - build new content
    1. Add Trino content to lakeFS-samples, update our Trino integration doc directly to reference it, update existing EB blog
    1. Add dbt content to lakeFS-samples, update our dbt integration doc directly to reference it, update existing EB blog

The risk of doing this in two stages is minimal given that very few people will find their way to the existing Everything Bagel for dbt and Trino since it's not referenced anywhere other than in passing in the EB blog (41 page views in the last 90 days)

**Comment by rmoff** [comment]
Zero feedback on this when shared [on Slack](https://lakefs.slack.com/archives/C014RNSB8GY/p1686820981867399) so going ahead with my proposal as planned.
